jira/client.py

```diff
@@ -2598,7 +2598,7 @@ def search_assignable_users_for_issues(self,
         :rtype: ResultList
         """
         params = {
-            'username': username}
+            'query': username}
```

Comment:
Does this work with super old versions of self-hosted JIRA?

Reply:
It wasn't causing an error but it was actually returning all the users and not really doing a match. Pushed a new fix.